app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -216,8 +217,10 @@ class FeedFragment : BaseStateFragment<FeedState>() {
         activity.supportActionBar?.subtitle = groupName
 
         inflater.inflate(R.menu.menu_feed_fragment, menu)
-        updateTogglePlayedItemsButton(menu.findItem(R.id.menu_item_feed_toggle_played_items))
-        updateToggleFutureItemsButton(menu.findItem(R.id.menu_item_feed_toggle_future_items))
+        MenuItemCompat.setTooltipText(
+            menu.findItem(R.id.menu_item_feed_toggle_played_items),
+            getString(R.string.feed_show_hide_streams)
+        )
```

Comment:
I don't think you need this. Since you never change it anymore, the tooltip text automatically corresponds to the title set in the XML.